multimedia/mlt/Portfile

```diff
@@ -80,100 +77,54 @@ depends_lib-append  port:atk \
                     port:libexif \
                     port:fftw-3 \
                     port:libebur128 \
-                    port:rubberband
+                    port:rubberband \
+                    port:rtaudio \
+                    port:libmovit
```

Comment:
Maybe the movit module and dependency should be a variant because its OpenGL usage can pull in some undesirable dependencies. One of these is `libsdl2`, which can be disabled by a variant, and that complicates this. But another is `mesa` that depends on:
```
...
  xorg-xorgproto
...
  xorg-libxcb
    xorg-libXau
    xorg-libXdmcp
    xorg-xcb-proto
      libxml2
        icu
  xorg-libX11
    xorg-xtrans
    xorg-util-macros
  xorg-libXext
    libtool
  xorg-libXdamage
    xorg-libXfixes
...
```

Reply:
These are mostly brought in with `cairo` as well via `frei0r-plugins`, I don't think it's an issue. `libsdl2` is also brought in by `ffmpeg6` so that's fine.

Reply:
`port rdeps frei0r-plugins` does not show me any xorg dependencies.

Reply:
[`frei0r-plugins`](https://ports.macports.org/port/frei0r-plugins/details/) -> `cairo` -> xrender -> xorg

Reply:
`cairo` only requires that when you enable the `opengl` variant.

Reply:
It's from the +x11 variant which is enabled by default. Either way I can remove the `libmovit` dependency if it's not desired.

Reply:
Oh, OK I have disabled `x11` in my macports config. Um, I do not feel strong about it, and I can see how it is desirable to have fewer variants as many users are not checking these when installing. It seems most variants here are to handle situations that have arisen over time.